### PR TITLE
Configurator - Validate schema

### PR DESCRIPTION
**Pull request by pkharat**

### CODE REVIEW

**Comment by pkharat** [comment]
@mak-454  Please review.

File: gbpservice/nfp/configurator/lib/schema.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+request_data_info = {'version': "",
+                     'service_type': ""
+                     }
+
+request_data_config = {'resource': "",
+                       'kwargs': ""
+                       }
+
+interfaces = {'context': {},
+              'request_info': {},
```
Comment (pkharat) [inline gbpservice/nfp/configurator/lib/schema.py]: Does config_agent add 'request_info' to kwargs ?

Comment (njagadish) [inline gbpservice/nfp/configurator/lib/schema.py]: As discussed, it is added by orchestrator.

**Comment by njagadish** [comment]
Merging this PR. Review is pending. Integration testing will continue in parallel with code review.

**Comment by njagadish** [comment]
Closing this PR as this is pushed using a PR #107

**Pull request closed without merging**